earth_enterprise/docker/README.md

```diff
@@ -156,10 +168,15 @@ If you want to build the Docker image yourself (perhaps, after customization),
 you can use the commands that follow.
```

Comment:
I think this is still confusing. We should put the instructions for Build Build Environment first, then Build Image, then Run Image